documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -14,13 +14,49 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 .Procedure
 
-. Upgrade the {operator-name}.
+Before upgrading, you must restart the CatalogSource pod.
+
+To restart the `CatalogSource` pod, do the following:
+
+.  In the the {ocp-short} web console, navigate to the *Operators* -> *Installed Operators* -> *Operator details* page. On the *Subscriptions* tab, note the name in the *CatalogSource* field.
```

Comment:
You do not need this step because the CLI step finds the pod.

Reply:
@apinnick 
The CLI step only gives a list of pods. The user has know which one to delete.

Reply:
I agree with @sbeskin-redhat 
It could be that there is more than one catalog source that can be used to install MTV. The user should know which catalog source was used and delete its pod